### PR TITLE
add conformance classes properties

### PR DESCRIPTION
**Pull request by maxcollombin**

### CODE REVIEW

File: core/sections/clause_10_labeling.adoc [hunk 1]
```diff
@@ -16,8 +16,23 @@ Base Classes: MultiGraphic: Graphic
 
 ===== Class FontOutline
 
+* size: float
+* opacity: float
+* color: Color
+
 ===== Class Font
-* outline property
+
```
Comment (jerstlouis) [inline core/sections/clause_10_labeling.adoc:24]: Here we just want to add _outline_ only. The Font class is already described in 8-vector.

File: core/sections/clause_13_advanced_strokes.adoc [hunk 2]
```diff
@@ -23,6 +31,10 @@ dashPattern property
 
 ===== Class Color
 
+* r: float
```
Comment (jerstlouis) [inline core/sections/clause_13_advanced_strokes.adoc:34]: Here we just want to use Color to type any new property. The Color class is already defined in 8-vector and 9-coverage.

File: core/sections/clause_8_vector.adoc [hunk 3]
```diff
@@ -7,6 +7,7 @@
 * fill:     Fill [0..1]
 * stroke:   Stroke [0..1]
 * marker:   Marker [0..1]
+* label:    Label [0..1]
```
Comment (jerstlouis) [inline core/sections/clause_8_vector.adoc:10]: label should not be here. It is added in 10-labeling Labeling conformance class.

File: core/sections/clause_8_vector.adoc [hunk 4]
```diff
@@ -18,25 +19,44 @@
...
 
-==== Class Stroke
-===== Class GraphicalUnit
-===== Class StrokeStyling
+==== Class Graphic
 
-==== Class MultiGraphic
-===== Class Marker
+* opacity: float
+* transform: Transform2D
```
Comment (jerstlouis) [inline core/sections/clause_8_vector.adoc:25]: We need to review _transform_ here. There was discussion that only _transform.translation_ would be supported here.

File: core/sections/clause_8_vector.adoc [hunk 5]
```diff
@@ -18,25 +19,44 @@
...
+
+====== Class Marker
+
+==== Class Resource
+
+==== Class Pointf    
+==== Class GraphicalUnit
+==== Class StrokeStyling
+==== Class Stroke
+==== Class TextAlignment
```
Comment (jerstlouis) [inline core/sections/clause_8_vector.adoc:57]: This undoes some fixes that were done: Moving TextAlignment and Font class inside Text section, and HAlignment / VAlignment being enum.

File: core/sections/clause_8_vector.adoc [hunk 6]
```diff
@@ -18,25 +19,44 @@
...
+==== Class HAlignment
+==== Class VAlignment
+==== Class Font
+
+The text property will exceptionnally support an identifier parameter value expression.
```
Comment (jerstlouis) [inline core/sections/clause_8_vector.adoc:62]: This should all be moved under the Text section.

File: core/sections/clause_8_vector.adoc [hunk 7]
```diff
@@ -18,25 +19,44 @@
...
+* elements: Graphic [0..*]
+
+====== Class Marker
+
+==== Class Resource
+
+==== Class Pointf    
+==== Class GraphicalUnit
+==== Class StrokeStyling
+==== Class Stroke
```
Comment (jerstlouis) [inline core/sections/clause_8_vector.adoc:56]: We are missing the Stroke properties here (but only the fact that it inherits from StrokeStyling with a color and width property / graphical unit).
We want to explain that Marker inherits from MultiGraphic.

Pointf is in the context of Dot point property, so we can describe under that section.

File: core/sections/clause_19_3D.adoc [hunk 8]
```diff
@@ -3,12 +3,35 @@
...
 
+* x: double
+* y: double
+* z: double
+
 ==== Class Vector3Df
 
+x: float
+y: float
+z: floa
```
Comment (jerstlouis) [inline core/sections/clause_19_3D.adoc:18]: missing a t

File: core/sections/clause_8_vector.adoc [hunk 9]
```diff
@@ -18,25 +19,44 @@
...
+===== Class Text
+
+* text: string
+* font: Font
+* alignment: TextAlignment
 
 ===== Class Shape
+
 ====== Class Dot
-* point: Pointf
```
Comment (jerstlouis) [inline core/sections/clause_8_vector.adoc:42]: Should not remove this here.

File: core/sections/clause_8_vector.adoc [hunk 10]
```diff
@@ -18,25 +19,44 @@
...
-====== Class Font
-====== Class TextAlignment
+* image: Resource
+* hotSpot: Pointf
+* tint: Color
+* blackTint: Color
+* alphaThreshold: float
 
-* enum HAlignment
-* enum VAlignment
```
Comment (jerstlouis) [inline core/sections/clause_8_vector.adoc:38]: We want to keep HAlignment / VAlignment as enum inside TextAlignment section.

File: core/sections/clause_8_vector.adoc [hunk 11]
```diff
@@ -18,25 +19,44 @@
...
-==== Class MultiGraphic
-===== Class Marker
+* opacity: float
+* transform: Transform2D
 
-==== Class Graphic
 ===== Class Image
-====== Class Resource
-===== Class Text
-The text property will exceptionnally support an identifier parameter value expression (without any additional conformance class required).
```
Comment (jerstlouis) [inline core/sections/clause_8_vector.adoc:32]: This needs to stay under text, as details of the `text` property.

File: core/sections/clause_8_vector.adoc [hunk 12]
```diff
@@ -18,25 +19,44 @@
...
 ====== Class Dot
-* point: Pointf
+
+===== Class MultiGraphic
+
+* elements: Graphic [0..*]
+
+====== Class Marker
+
+==== Class Resource
```
Comment (jerstlouis) [inline core/sections/clause_8_vector.adoc:51]: Resource should be under the Image section where it is used, with its properties.

File: core/sections/clause_16_functions.adoc [hunk 13]
```diff
@@ -11,69 +11,184 @@
...
+* lineStrings: LineString[1..*]
+
 ===== Class PolygonContour
 
+* points: Point[3..*]
+
 ===== Class Polygon
+
+* outer: PolygonContour
+inner: PolygonContour[0..*]
```
Comment (jerstlouis) [inline core/sections/clause_16_functions.adoc:190]: This is missing a bullet.

File: core/sections/clause_16_functions.adoc [hunk 14]
```diff
@@ -11,69 +11,184 @@
...
 ==== Overview
 
 ===== Standard GeometryManipulationFunctions
 
+* s_intersection(Geometry a, Geometry b) : Geometry
+* s_union(Geometry a, Geometry b) : Geometry
+* s_subtraction(Geometry a, Geometry b) : Geometry
+* s_buffer(Geometry a, double d) : Geometry
+
 ===== Class Geometry
```
Comment (jerstlouis) [inline core/sections/clause_16_functions.adoc:160]: It looks like we have all the Geometry stuff twice?

We can have a section for it before both the Geometry Manipulation and Geometry relation section. Shouldn't be duplicated.

File: core/sections/clause_16_functions.adoc [hunk 15]
```diff
@@ -11,69 +11,184 @@
...
+* lineStrings: LineString[1..*]
+
 ===== Class PolygonContour
 
+* points: Point[3..*]
+
 ===== Class Polygon
 
+* outer: PolygonContour
+inner: PolygonContour[0..*]
```
Comment (jerstlouis) [inline core/sections/clause_16_functions.adoc:62]: Missing a bullet.

**Review by jerstlouis — CHANGES_REQUESTED** [review]
Thank you @maxcollombin for this! A few things to fix please, especially in 8-vector section.

File: core/sections/clause_16_functions.adoc (hunk 14 above)
Comment (maxcollombin) [reply core/sections/clause_16_functions.adoc:160]: This redundant class has been deleted.

File: core/sections/clause_16_functions.adoc (hunk 15 above)
Comment (maxcollombin) [reply core/sections/clause_16_functions.adoc:62]: bullet added

File: core/sections/clause_19_3D.adoc (hunk 8 above)
Comment (maxcollombin) [reply core/sections/clause_19_3D.adoc:18]: The missing "t" has been added.

File: core/sections/clause_10_labeling.adoc (hunk 1 above)
Comment (maxcollombin) [reply core/sections/clause_10_labeling.adoc:24]: removed unnecessary/redundant properties

File: core/sections/clause_13_advanced_strokes.adoc (hunk 2 above)
Comment (maxcollombin) [reply core/sections/clause_13_advanced_strokes.adoc:34]: removed the associated properties

File: core/sections/clause_16_functions.adoc (hunk 13 above)
Comment (maxcollombin) [reply core/sections/clause_16_functions.adoc:190]: bullet added

File: core/sections/clause_8_vector.adoc (hunk 6 above)
Comment (maxcollombin) [reply core/sections/clause_8_vector.adoc:62]: moved under the appropriate section

File: core/sections/clause_8_vector.adoc (hunk 5 above)
Comment (maxcollombin) [reply core/sections/clause_8_vector.adoc:57]: moved under text

File: core/sections/clause_8_vector.adoc (hunk 3 above)
Comment (maxcollombin) [reply core/sections/clause_8_vector.adoc:10]: removed label property

**Pull request merged**